### ISSUE TITLE
"Pagination feature" Focus controls >| doesn't work with keyboard or mouse clicks

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#828]
Steps to reproduce:

Load [this page](https://maps4html.org/experiments/custom-projections/BNG/)
Turn on BNG Shrink/Swell WMS layer
Zoom to BNG Shrink/Swell WMS layer (right-click on layer in layer control, select Zoom To Layer)
Tab into map or click on map on blue BNG Shrink/Swell WMS layer to display query response:

![image](https://user-images.githubusercontent.com/2437285/233158499-0ef1c7b5-8a86-469c-bb88-5868c93dd6e0.png)

Click on ">|" or tab to it and hit enter/spacebar, 

Expected behaviour:

Zoom control should receive focus

Observed behaviour:

Nothing happens

**Comment by prushforth** [comment]
On map-link branch, there's an error message in the console for this bug: 

![image](https://github.com/Maps4HTML/Web-Map-Custom-Element/assets/2437285/ea276c5f-4721-4d6d-bcd6-10ce7c60a815)
